pkg/kobject/kobject.go

```diff
@@ -99,8 +98,20 @@ type ServiceConfig struct {
 	Dockerfile      string              `compose:"dockerfile"`
 	Replicas        int                 `compose:"replicas"`
 	GroupAdd        []int64             `compose:"group_add"`
-	// Volumes is a struct which contains all information about each volume
-	Volumes []Volumes `compose:""`
+	Volumes         []Volumes           `compose:""`
+	HealthChecks    HealthCheck         `compose:""`
+}
+
+// HealthCheck the healthcheck configuration for a service
+// "StartPeriod" is not yet added to compose, see:
+// https://github.com/docker/cli/issues/116
+type HealthCheck struct {
+	Test        []string
+	Timeout     int32
+	Interval    int32
+	Retries     int32
+	StartPeriod int32
+	Disable     bool
```

Comment:
Are we doing anything with this field in code, I couldn't find anything.
Should we not add a check that if this is false, make sure that no Kubernetes probes are generated?